roles/gdm/meta/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+dependencies:
+  - { role: systemd }
```

Comment:
This is only necessary when you need to notify the systemd reload handler after installing or modifying a service or timer file (that's the only thing the systemd role provides).